examples/example4.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from kubernetes import client, config
+from kubernetes.client import configuration
+
+
+def main():
+    config_file = os.environ["HOME"] + '/.kube/config'
+    contexts, active_context = config.list_kube_config_context(config_file)
+    if not contexts:
+        print("Cannot find any context in kube-config file.")
+        return
+    for i in range(len(contexts)):
```

Comment:
The reason is a deleted part of the example that show hostname of the selected context. But even for that, it can be done easier. I will fix the example tomorrow. Thanks.